components/application-operator/charts/application/templates/validator.yaml

```diff
@@ -37,6 +37,8 @@ spec:
             - "--eventServicePathPrefixV1=/{{ .Release.Name }}/v1/events"
             - "--eventServicePathPrefixV2=/{{ .Release.Name }}/v2/events"
             - "--eventServiceHost={{ .Release.Name }}-event-service:{{ .Values.eventService.service.externalapi.port }}"
+            - "--eventMeshPathPrefix=/{{ .Release.Name }}/events"
```

Comment:
Shouldn't it be added to the `VirtualService` here https://github.com/kyma-project/kyma/pull/6422/files#diff-4b481e57ffc75a94ee22ebc3f342cd94R16 ?